datafusion/functions-nested/benches/map.rs

```diff
@@ -96,6 +96,7 @@ fn criterion_benchmark(c: &mut Criterion) {
 
         b.iter(|| {
             black_box(
+                #[allow(deprecated)] // TODO use invoke_batch
```

Comment:
can we do `#[warn(deprecated)]` instead? My vision it should output the `note` in this case?

Reply:
i though we treat warnings as errors. will this work?